### PR TITLE
Trivial javalib methods

### PR DESCRIPTION
**Pull request by sjrd**

### CODE REVIEW

File: test-suite/src/test/scala/org/scalajs/testsuite/javalib/StringTest.scala [hunk 1]
```diff
@@ -149,6 +149,23 @@ object StringTest extends JasmineTest {
...
+      expect(s.codePointCount(2, 4)).toEqual(2)
+      expect(s.codePointCount(2, 5)).toEqual(2)
+      expect(s.codePointCount(2, 6)).toEqual(3)
+      expect(s.codePointCount(12, 17)).toEqual(5)
+      expect(s.codePointCount(8, 10)).toEqual(2)
+      expect(s.codePointCount(7, 10)).toEqual(2)
+      expect(s.codePointCount(7, 7)).toEqual(0)
+      expect(s.codePointCount(s.length - 1, s.length)).toEqual(1)
+      expect(s.codePointCount(s.length - 1, s.length - 1)).toEqual(0)
+      expect(s.codePointCount(s.length, s.length)).toEqual(0)
```
Comment (gzm0) [inline test-suite/src/test/scala/org/scalajs/testsuite/javalib/StringTest.scala]: Add a test for an exception case.

File: test-suite/src/test/scala/org/scalajs/testsuite/javalib/SystemTest.scala [hunk 2]
```diff
@@ -21,6 +21,39 @@ object SystemTest extends JasmineTest {
 
   describe("java.lang.System") {
 
+    it("setIn") {
+      val savedIn = System.in
+      try {
+        val testIn = new  java.io.ByteArrayInputStream(Array[Byte]())
```
Comment (gzm0) [inline test-suite/src/test/scala/org/scalajs/testsuite/javalib/SystemTest.scala]: Superfluous space

**Comment by gzm0** [comment]
That's all

**Comment by sjrd** [comment]
Updated.

**Comment by gzm0** [comment]
LGTM

**Pull request merged**